atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/async/AsyncCellLoader.java

```diff
@@ -82,12 +84,15 @@ private AsyncCellLoader(CqlClient cqlClient, Executor executor, String keyspace)
             long timestamp) {
         return cqlClient.executeQuery(
                 ImmutableGetQuerySpec.builder()
-                        .tableReference(tableRef)
-                        .keySpace(keyspace)
-                        .row(ByteBuffer.wrap(cell.getRowName()))
-                        .column(ByteBuffer.wrap(cell.getColumnName()))
-                        .humanReadableTimestamp(timestamp)
-                        .executor(executor)
+                        .cqlQueryContext(ImmutableCqlQueryContext.builder()
+                                .tableReference(tableRef)
+                                .keySpace(keyspace)
+                                .executor(executor).build())
+                        .queryParameters(ImmutableGetQueryParameters.builder()
+                                .row(ByteBuffer.wrap(cell.getRowName()))
+                                .column(ByteBuffer.wrap(cell.getColumnName()))
+                                .humanReadableTimestamp(timestamp)
+                                .build())
```

Comment:
AsyncCellLoader#getDone is a bit suspicious, I wouldn't think Cass would throw any checked exceptions in this codepath.

Reply:
This is a recommended way from Guava docs to work with `Futures#whenAllSucceed.call(...)`.

Reply:
unless the builder can be done in one line, please refrain from nesting builders.

Reply:
+1 to what @OStevan said, if we didn't, we'd also have to then deal with interrupted exception etc. less code 💃 , although part of me is tempted to do a `getDoneUnchecked` but it's probably a bad idea

Reply:
nit: I'm not sure where in the docs you're linking to - [whenAllSucceed](https://guava.dev/releases/20.0/api/docs/com/google/common/util/concurrent/Futures.html#whenAllSucceed-java.lang.Iterable-) or [FutureCombiner](https://guava.dev/releases/20.0/api/docs/com/google/common/util/concurrent/Futures.FutureCombiner.html#call-java.util.concurrent.Callable-), or something else.

I guess this is a refactor so fine, but the original seems a bit questionable in that it would wrap unchecked exceptions in another `RuntimeException` as well